tjpcov/covariance_clusters.py

```diff
@@ -0,0 +1,370 @@
+from .covariance_builder import CovarianceBuilder
+from .clusters_helpers import MassRichnessRelation, FFTHelper
+import numpy as np
+import pyccl as ccl
+from scipy.integrate import quad, romb
+
+
+class CovarianceClusters(CovarianceBuilder):
+    """The base class for calculating covariance that includes galaxy cluster
+    number counts.
+    """
```

Comment:
You need to specify the type of tracers that this class is able to compute the covariance for (i.e. whatever `sacc_file.tracers["name_of_tracer"].quantity` returns. Maybe the following?

```
_tracers_types = ("cluster", "cluster")
```

Reply:
I specified this on the concrete implementation (`ClusterCounts`) will leave it off of the base class as it may be different for N x C_ell

Reply:
True, I missed that.